Algorithm.CSharp/MeanVarianceOptimizationFrameworkAlgorithm.cs

```diff
@@ -104,6 +104,7 @@ public override void OnOrderEvent(OrderEvent orderEvent)
             {"Treynor Ratio", "3.236"},
             {"Total Fees", "$26.11"},
             {"Estimated Strategy Capacity", "$25000000.00"},
+            {"Lowest Capacity Asset", "AIG R735QTJ8XC9X"},
```

Comment:
Personally I don't like the inconsistency in the way we are showing this, basically cached symbols or not

Reply:
Went ahead and used Symbol.Value so we don't rely on the cache to be set for `ToString()` format to be consistent.

I chose this over Symbol.ID.ToString() because its behavior is unique if it has an underlying and isn't as easily identifiable as the Symbol.Value.

Ex. 
A GC Futures Contract is the "Lowest Capacity Asset"
Symbol.ID.ToString() -> GC VOFJUCDY9XNH
Symbol.Value -> GC26G14

ES Future Option Contract 
Symbol.ID.ToString() -> ES 31C3JQTOYO9T0|ES XCZJLC9NOB29
Symbol.Value -> ES20H20  200320P03150000

TWX Option
TWX (AOL Previously)
Symbol.ID.ToString() -> AOL VRKS95ENLBYE|AOL R735QTJ8XC9X
Symbol.Value -> TWX   140621C00042500

As you can see the `Value` is a lot more specific on exactly what contract or option you are dealing with. Then in the case of all equities it will use the newest ticker for that where as the SID will output the original listing ticker.